src/Stache/Repositories/EntryRepository.php

```diff
@@ -43,6 +44,17 @@ public function find($id): ?Entry
         return $this->query()->where('id', $id)->first();
     }
 
+    public function findOrFail($id): Entry|EntryNotFoundException
+    {
+        $result = $this->query()->where('id', $id)->first();
+
+        if (is_null($result)) {
```

Comment:
This can be simplified to:

```suggestion
        if (! $result) {
```

Reply:
most static analyzers prefer the `is_null` because it's "more correct".